content/webapp/components/OnThisPageAnchors/index.tsx

```diff
@@ -155,55 +205,96 @@ const OnThisPageAnchors: FunctionComponent<Props> = ({
     }
   }, [activeId]);
 
+  useEffect(() => {
+    if (!listRef.current || !isSticky) return;
+    listRef.current.classList.add('is-hidden-s');
+    listRef.current.classList.add('is-hidden-m');
+  }, [listRef.current]);
+
+  function toggleList() {
+    if (!listRef.current || !buttonRef.current) return;
+
+    if (listRef.current.classList.contains('is-hidden-s')) {
+      listRef.current.classList.remove('is-hidden-s');
+      listRef.current.classList.remove('is-hidden-m');
+
+      buttonRef.current.setAttribute('aria-expanded', 'true');
+    } else {
+      listRef.current.classList.add('is-hidden-s');
+      listRef.current.classList.add('is-hidden-m');
+      buttonRef.current.setAttribute('aria-expanded', 'false');
+    }
+  }
+
+  function hideMobileList() {
+    if (!listRef.current || !buttonRef.current) return;
+
+    listRef.current.classList.add('is-hidden-s');
+    listRef.current.classList.add('is-hidden-m');
```

Comment:
Would need to be tested but I think you can do
```suggestion
    listRef.current.classList.add('is-hidden-s', 'is-hidden-m');
```

Reply:
Could toggleList() be reused? They do similar things?

Reply:
I didn't think I could usefully reuse `toggleList` without it become more convoluted/confusing (I also experimented with using `classList.toggle` which has an optional `force` param, but again I thought this seemed clearer). Adding/removing multiple classes is neat though – think I've always thought that wasn't possible 🤦‍♂️